### PR TITLE
Fix build with GHC 8 by bumping bounds

### PR DESCRIPTION
**Pull request by spwhitton**
This patch courtesy of Clint Adams.

### CODE REVIEW

File: token-bucket.cabal [hunk 1]
```diff
@@ -27,19 +27,19 @@ flag use-cbits
...
   default-language:    Haskell2010
-  build-depends:       base >=4.6 && <4.9
+  build-depends:       base >=4.6 && <4.10
   hs-source-dirs:      src
   if flag(use-cbits)
     cpp-options:       -DUSE_CBITS
     c-sources:         cbits/get_posix_time.c
   else
-    build-depends:     time >=1.4 && <1.6
+    build-depends:     time >=1.4 && <1.8
```
Comment (hvr) [inline token-bucket.cabal]: The upper bound on `time` seems wrong; there is no `time-1.7` or later release afaik

File: token-bucket.cabal [hunk 2]
```diff
@@ -27,19 +27,19 @@ flag use-cbits
...
   ghc-options: -Wall
   exposed-modules:     Control.Concurrent.TokenBucket
 
 test-suite test-tb
   default-language:    Haskell2010
   type:                exitcode-stdio-1.0
-  build-depends:       base >=4.6 && <4.9, time >=1.4 && <1.6, token-bucket
+  build-depends:       base >=4.6 && <4.10, time >=1.4 && <1.8, token-bucket
```
Comment (hvr) [inline token-bucket.cabal]: likewise

**Comment by hvr** [comment]
resolved via 36c2cf9df4e6668d7f02dfbb93fff5877c742f53

**Comment by spwhitton** [comment]
Thank you for 36c2cf9!

**Pull request closed without merging**